db/schema.rb

```diff
@@ -212,6 +212,7 @@
     t.boolean "temp_reuse_applicant_email"
     t.boolean "temp_reuse_operator_address"
     t.string "temp_reuse_address_for_site_location"
+    t.boolean "temp_use_registered_company_details"
```

Comment:
Curious why the schema was updated?

Reply:
not to sure - this happens whenever I `vagrant up` will remove!